codice/ddf
# DDF-4594 Added ability to perform an action on multiple selected items

#### What does this PR do?
Creates a section above the visualization tabs for buttons that perform actions on multiple selected items. Currently the only button is "Export"
#### Who is reviewing it? 
@andrewzimmer @Bdthomson 
#### Select relevant component teams: 
#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@bdeining 
@djblue 
#### How should this be tested?
- Verify that an `Export` button exists over the visualization tabs and is disabled with no results selected
- Select at least one result and verify that the `Export` button is enabled
- Click the `Export` button and verify that the selected result(s) are properly exported
#### Any background context you want to provide?
#### What are the relevant tickets?
Fixes: #4594 
#### Screenshots
![DDF-4594-min (1)](https://user-images.githubusercontent.com/39737329/55186271-c164e700-515b-11e9-855e-71eff3cb1fd1.gif)
#### Checklist:
#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
